maintainers/maintainer-list.nix

```diff
@@ -184,6 +184,12 @@
     githubId = 1250775;
     name = "Adolfo E. García Castro";
   };
+  adsr = {
```

Comment:
The addition to maintainers.nix should be in a separate commit titled "maintainers: add adsr"